naive_bayes.py

```diff
@@ -135,20 +135,21 @@ def classify_recipe_by_ingredients(ingred_dict, link, recipe):
     primary_methods = ["boil", "bake", "brown", "cook", "deep-fry", "stir-fry", "simmer", "baste", "roast", "grill", "broil", "pan-fry", "poach", "steam", "braise", "stew", "scald", "sear", "blanch", "barbeque", "griddle", "sear", "fry", "melt"]
     secondary_methods = ["chop", "stir", "beat", "cream", "cure", "dice", "drizzle", "fold", "glaze", "julienne", "marinate", "mince", "sear", "shred", "sift", "slice", "peel", "puree", "reduce", "grate", "deglaze", "season", "crush", "squeeze", "shake"]
 
     recipe_to_cuisine = []
 
-    with open('saved_crawlr.json') as f:
+    with open(saved_file) as f:
         recipe_map = json.load(f)
         for link, recipe in recipe_map.iteritems():
 
-            recipe_to_cuisine.append(
-                classify_recipe_by_ingredients(ingred_dict, link, recipe))
-            arecipe = classify_recipe_by_ingredients(ingred_dict, link, recipe)
+            # Tom: commented this out, not sure if it should be here as it creates duplicate recipe
```

Comment:
Nope this is not needed.

On Thu, Mar 12, 2015 at 4:11 PM, Sen Lu Tom Chen <notifications@github.com>
wrote:

> In naive_bayes.py
> <https://github.com/grantnicholas/all_recipes/pull/1#discussion_r26344898>
> :
>
> >          recipe_map = json.load(f)
> >          for link, recipe in recipe_map.iteritems():
> >
> > -            recipe_to_cuisine.append(
> > -                classify_recipe_by_ingredients(ingred_dict, link, recipe))
> > -            arecipe = classify_recipe_by_ingredients(ingred_dict, link, recipe)
> > +            # Tom: commented this out, not sure if it should be here as it creates duplicate recipe
>
> @grantnicholas <https://github.com/grantnicholas> can you double check if
> this should be here or not? It was creating duplicate recipe in the json,
> not sure if that was intended
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grantnicholas/all_recipes/pull/1/files#r26344898>.
>